OlinkAnalyze/tests/testthat/test-pca_plot.R

```diff
@@ -20,6 +20,19 @@ pca_plot_treatCol_topLoadings <- npx_data1 %>%
                      head(5) %>%
                      pull(OlinkID)})
 
+#PCA by panel
```

Comment:
I think thie tests look fine but may be worth having someone else double check, Im not really familar with tests